test/mocks/values/invoice-number.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
   LUMP_SUMS_INVOICE_NUMBER: 'LSES0000001',
   CS_INVOICE_NUMBER: 'CS000000001',
   BPS_INVOICE_NUMBER: 'SITI0000001',
-  FDMR_INVOICE_NUMBER: 'FDMR0000001'
+  FDMR_INVOICE_NUMBER: 'FDMR0000001',
+  MANUAL_INVOICE_NUMBER: 'S1234567S1234567V001'
```

Comment:
The format for manual invoices varies, but here's an example you can use here that would make the mock more realistic.

`SPS 2014_3VKYT05`